libass/ass_fontconfig.c

```diff
@@ -92,6 +157,9 @@ static void scan_fonts(FcConfig *config, ASS_FontProvider *provider)
     // get list of fonts
     fonts = FcConfigGetFonts(config, FcSetSystem);
 
+    // temporary workaround: sort list to prefer regular variants
```

Comment:
Not sure how useful this comment is. Might be better either to paste the explanation from the commit message (perhaps shortened, perhaps not) or to remove the “temporary workaround” part.

Reply:
dropped "temporary workaround"